jdaviz/configs/cubeviz/plugins/moment_maps/moment_maps.py

```diff
@@ -103,10 +101,6 @@ def _on_subset_selected(self, event):
             self.spectral_min = spec_sub.lower.value
             self.spectral_max = spec_sub.upper.value
 
-    @observe("filename")
```

Comment:
I don't understand why this `observe` was even necessary. I can add it back if it was there for a reason.